css/main.sass

```diff
@@ -528,4 +539,6 @@ footer ul
         display: block
         width: 100%
     nav
-        height: 100%
\ No newline at end of file
+        height: 100%
+    .social
+        display: block !important
```

Comment:
Avoid !important; simply move this to where the other styling is being applied

Reply:
Please can you explain?